src/main/java/edu/wpi/first/wpilibj/networktables/NetworkTableValue.java

```diff
@@ -0,0 +1,325 @@
+package edu.wpi.first.wpilibj.networktables;
+
+import java.util.Objects;
+
+/**
+ * A network table entry value.
+ */
+public class NetworkTableValue {
+  public NetworkTableValue(NetworkTableType type, long lastChange, Object value) {
```

Comment:
They were private originally.  I'm not sure why @AustinShalit wanted them public.